### DISCUSSION TITLE
Trouble with second dhcp handshake on I211AT /Asus gamer boards

### DISCUSSION THREAD

**Discussion by BillyCroan** [ipxe/ipxe#1256] [General]
[74a7.cap.zip](https://github.com/user-attachments/files/16200342/74a7.cap.zip)
I have an Asus PRIME TRX40-Pro board with one onboard intel I211AT gig phy.
![image](https://github.com/user-attachments/assets/c564fbaa-86a5-4701-be3e-878e741ae1cb)
It receives my compiled undionly.kpxe over tftp just fine after its first dhcp performed by the nic itself.  But then I believe tht kpxe "kernel" boots and does its own dhcp negotiation.  that second one fails apparently:
![image](https://github.com/user-attachments/assets/bf315eef-e703-48e3-a2c1-be44bf3dc62c)
I followed https://ipxe.org/28086011 and already tried updating to latest source, and adding netdev to my DEBUG. 

Also.  I know that a second dhcp failing reeks of portfast.  but I already added "spanning-tree 802-1w admin-edge-port" to my port.  I also have not generally needed that elsewhere on dozens of other systems.  This is the only TRX40-Pro system however and boy do I regret this motherboard.

I saved a dump of dhcp events in syslog and some screenshots at https://docs.google.com/document/d/1TafbTG_XZvq4Yh8qieHdY9g8XYW1dA_Qdpk3JnUOAfI/edit and I also have a pcap if interested.

What else should I try?

**Comment by NiKiZe** [comment]
Enter iPXE shell, primarily with default build using Ctrl-B
Run `ifstat`, `dhcp`, `ifstat`

Do you have any `RX` packets at all? If you have `TX` try to grab that with a packet capture, preferably by having something connected directly to the NIC.

**Comment by BillyCroan** [comment]
no RX packets. at all.    It TX'd 7 packets, and it had one TXE even after a fresh boot for whatever that means.  is that TXE normal?

I slept 60 seconds and retried dhcp and ifstat shows 14 tx abd no rx:
![image](https://github.com/user-attachments/assets/264c37ce-7bf1-4ab7-8111-5193fafb5a7c)

**Comment by ymolinet** [comment]
HI all,

I have the same issue with a fresh compiled ipxe undionly.kpxe from source on a motherboard containing the same network chipset.
Do you have solved this ? and how ?

Thanks,

**Reply by NiKiZe** [reply]
Any particular reason you are using pcbios mode instead of EFI? And if you where to try EFI, does the behavior change in any way?

**Reply by ymolinet** [reply]
Just because I'm using a little appliance that is only bios mode. This appliance could not boot efi image.
